test/typescript/renderHook.ts

```diff
@@ -0,0 +1,60 @@
+import { useState, createContext, useContext, useMemo } from 'react'
+import { renderHook } from 'react-hooks-testing-library'
+
+const DARK: 'dark' = 'dark'
+const LIGHT: 'light' = 'light'
+
+type InitialTheme = typeof DARK | typeof LIGHT | undefined
+
+const themes = {
+  light: { primaryLight: '#FFFFFF', primaryDark: '#000000' },
+  dark: { primaryLight: '#000000', primaryDark: '#FFFFFF' }
+}
+
+const ThemesContext = createContext(themes)
+
+const useTheme = (initialTheme: InitialTheme = DARK) => {
```

Comment:
Add minimum typing tests with the sample code in README.md.

Reply:
I'm not a big fan of that example, but I wanted something that used a few different hooks, so it became a little contrived.  Happy for you to come up with something better if you want, or just keep it as is that's easier.

Reply:
> so it became a little contrived

I see.

> Happy for you to come up with something better

I'll give it a try soon 👍

Reply:
Replace `useTheme` hook with `useCounter` here 3cb8f92.
Also fix type definitions because I just realized they are broken.